composer.json

```diff
@@ -219,7 +219,7 @@
         "spryker/sales-returns-rest-api": "^1.1.0",
         "spryker/sales-statistics": "^1.2.0",
         "spryker/scheduler-jenkins": "^1.2.1",
-        "spryker/search-http": "^0.1.0",
+        "spryker/search-http": "^0.3.1",
```

Comment:
https://github.com/spryker/suite-nonsplit/pull/7744/files
but requires SearchHttp upgrade 0.3.1 >> 0.3.2


cc @Maksym-Ohnianikov

Reply:
Updated
Do we want another QA round?